core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -199,9 +198,14 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
         .registerContributorDashboardViewSuggestionForReview('Translation');
       this.heading = 'Review Translation Contributions';
     }
+    const suggestionIds = Object.keys(this.suggestionIdToContribution);
+    const clickedSuggestionIndex = suggestionIds.indexOf(
+      this.activeSuggestionId);
+    this.skippedContributionIds = (
+      [...suggestionIds.slice(0, clickedSuggestionIndex)]);
```

Comment:
```suggestion
      suggestionIds.slice(0, clickedSuggestionIndex)];
```
`.slice()` already returns a new Array, so I don't think the spread operator is necessary here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice.